### PR TITLE
refactor: rename landing page components and remove atoms from storybook

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

rename landing page components to not include "section" in name and remove landing page atom storybook pages

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/2aBlW1pgFt5PT6soRu1HXy/%E2%9C%8F%EF%B8%8F-Landing-Page-Templates---Unification?node-id=1%3A43
- [x] Related to JIRA ticket: [REACH-432]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@6.0.2-alpha.33cc95.0<br/>@codecademy/styleguide@19.0.2-alpha.33cc95.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc558c12d15a40b610321d2--gamut-preview.netlify.app>https://5fc558c12d15a40b610321d2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc558c12d15a40b610321d2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@6.0.2-alpha.d47bdb.0<br/>@codecademy/styleguide@19.0.2-alpha.d47bdb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc559cb47a17900ff4e9cc7--gamut-preview.netlify.app>https://5fc559cb47a17900ff4e9cc7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc559cb47a17900ff4e9cc7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@6.0.2-alpha.ced8f5.0<br/>@codecademy/styleguide@19.0.2-alpha.ced8f5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc55f432d15a40cbb03273a--gamut-preview.netlify.app>https://5fc55f432d15a40cbb03273a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc55f432d15a40cbb03273a>Deploy Logs</a></p>

**Review by JoshuaKGoldberg — APPROVED** [review]

File: packages/gamut-labs/src/landingPage/Feature.tsx [hunk 1]
```diff
@@ -62,10 +62,10 @@ export type PageFeatureProps = {
   testId?: string;
 };
 
-export const PageFeature: React.FC<PageFeatureProps> = ({
-  isIcon = false,
+export const Feature: React.FC<FeatureProps> = ({
+  isIcon,
   imgSrc,
-  imgAlt,
+  imgAlt = '',
```
Comment (rogervera) [inline packages/gamut-labs/src/landingPage/Feature.tsx:68]: 👍

**Review by rogervera — APPROVED** [review]
LGTM

**Comment by christian-dinh** [comment]
this is it 🔥

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@6.0.4-alpha.d05c99.0<br/>@codecademy/styleguide@20.0.1-alpha.d05c99.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fc67553ddfbcd0e83908b57--gamut-preview.netlify.app>https://5fc67553ddfbcd0e83908b57--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fc67553ddfbcd0e83908b57>Deploy Logs</a></p>

**Pull request merged**